### ISSUE TITLE
Social Flow icon not visible when scrolled to bottom of page in dark mode

### ISSUE THREAD

**Issue by DasAnurag31** [girlscript/gssoc-website-new#114]
<img width="80" alt="image" src="https://github.com/girlscript/gssoc-website-new/assets/92816250/91db1793-0c9d-4638-ab6b-b4e37c6fc9e9">

**Comment by DasAnurag31** [comment]
Can i be assigned this issue ?

**Comment by Suvraneel** [comment]
Seems fine. What device are you using? @DasAnurag31 
![image](https://github.com/girlscript/gssoc-website-new/assets/63473496/89e9a5fc-df80-4bde-9112-e9fc88328321)

**Comment by Suvraneel** [comment]
@DasAnurag31  Cool found it...
![image](https://github.com/girlscript/gssoc-website-new/assets/63473496/7eeb8102-097d-402a-927e-fa818a6c09a5)
Yeah go ahead. Do not add any background or anything to socials... just make the footer responsive to smaller viewports(ie, >phone(xs,sm) but <~iPad(md))

Also, try to use tailwind queries only instead of stock CSS unless absolutely required

**Comment by Aman-Manwani** [comment]
@DasAnurag31 Are you still doing this ??

**Comment by Aman-Manwani** [comment]
@Suvraneel I think it's better to change socials color to white

**Comment by DasAnurag31** [comment]
Yes. 
I tried out some changes with the background. 
I also tried out white, but it did not matchthe colour palette

**Comment by DasAnurag31** [comment]
Changing the size of the footer or making it responsive isn't helping. I think we should change the colour of the icons to resolve the issue

**Comment by Suvraneel** [comment]
> Yes. I tried out some changes with the background. I also tried out white, but it did not matchthe colour palette

I agree but changing the colors isn't an option. What can be done is either to make the icons sticky uptill that point where the footer starts OR (a more cool but difficult approach) the orange colored icon can morph to black once it goes on top of footer.

Either way lemme know if any one of you two wanna take it up

**Comment by abhinav2712** [comment]
> Changing the size of the footer or making it responsive isn't helping. I think we should change the colour of the icons to resolve the issue

I feel like its matching:
![image](https://github.com/girlscript/gssoc-website-new/assets/68495520/266427a6-fc38-4248-91c5-daf16e1a441c)

**Comment by abhinav2712** [comment]
or what we can do it to add a slider background but someone earlier said not to..

**Comment by gurjeetsinghvirdee** [comment]
#131
